astropy/visualization/wcsaxes/core.py

```diff
@@ -307,25 +332,179 @@ def plot_coord(self, *args, **kwargs):
 
             plot_data = []
             for coord in self.coords:
-                if coord.coord_type == 'longitude':
+                if coord.coord_type == "longitude":
                     plot_data.append(frame0.spherical.lon.to_value(u.deg))
-                elif coord.coord_type == 'latitude':
+                elif coord.coord_type == "latitude":
                     plot_data.append(frame0.spherical.lat.to_value(u.deg))
                 else:
-                    raise NotImplementedError("Coordinates cannot be plotted with this "
-                                              "method because the WCS does not represent longitude/latitude.")
+                    raise NotImplementedError(
+                        "Coordinates cannot be plotted with this "
+                        "method because the WCS does not represent longitude/latitude."
+                    )
 
-            if 'transform' in kwargs.keys():
-                raise TypeError("The 'transform' keyword argument is not allowed,"
-                                " as it is automatically determined by the input coordinate frame.")
+            if "transform" in kwargs.keys():
+                raise TypeError(
+                    "The 'transform' keyword argument is not allowed,"
+                    " as it is automatically determined by the input coordinate frame."
+                )
 
             transform = self.get_transform(native_frame)
-            kwargs.update({'transform': transform})
+            kwargs.update({"transform": transform})
 
             args = tuple(plot_data) + args[1:]
 
         return super().plot(*args, **kwargs)
 
+    def add_beam(
+        self,
+        header=None,
+        major=None,
+        minor=None,
+        angle=None,
+        corner="bottom left",
+        frame=False,
+        borderpad=0.4,
+        pad=0.5,
+        **kwargs,
+    ):
```

Comment:
I would prefer to keep these as helper functions that don't live as methods on the class, because they only apply to specific kinds of data (for instance only celestial WCS) - I think we could develop a whole set of helper functions for different kinds of WCS and would prefer not to add too much to the main WCSAxes class. So could you add these to a new ``astropy.visualization.wcsaxes.helpers`` submodule as functions and import them into the ``astropy.visualization.wcsaxes`` namespace?